res/skins/Traktmixxx/license.txt

```diff
@@ -0,0 +1,26 @@
+<!--
+"Traktmixxx", Skin for Mixxx
+www.mixxx.org
+Copyright (C) 2015-2020 Inti Alonso (https://github.com/intialonso), built over "Roundcorners", the amazing work of 22degrees (https://github.com/22degrees)
```

Comment:
"RoundCorners" and it should include the full URL to the GitHub repository. Adding an additional link to the forum post https://www.mixxx.org/forums/viewtopic.php?f=8&t=9443 doesn't hurt.

I would suggest to separate your own copyright state from the attribution.

Reply:
Just signed the agreement. 
Im reading about the newlines thing, Im working on Windows, how can I batch fix the issue?
Regarding the license do you mean just puting another line? Like this:

> Copyright (C) 2015-2020 by Inti Alonso https://github.com/intialonso
Built over "RoundCorners", by 22degrees https://github.com/22degrees/RoundCorners https://www.mixxx.org/forums/viewtopic.php?f=8&t=9443

Im sorry if everything is a little messed, Im more a designer than a programer, so a lot of details confuse me.